### DISCUSSION TITLE
I have created a route, but its broken on screen

### DISCUSSION THREAD

**Discussion by SebaFR7474** [Lailloken/Exile-UI#389] [Q&A] [answered]
Hi, i have created a route, and exported in and imported un lailloken, but its only showing all the text, and not from the begining, from the act.

How i can show you or give the route to check? or what do we need to check when we edit route ? thx

**Comment by Lailloken** [comment]
You can upload a screenshot of the overlay. I have never used an edited/custom route, but I suspect it can break if you leave out area transitions. The tool simply fills the current page with guide-steps until there's an "enter XYZ" step. If you enter that zone with your character, the tool "knows" it can switch to the next page and fill that again until it hits the next transition.

There's no other way to have the overlay adapt to what you're doing.

**Comment by SebaFR7474** [comment]
here even from act 1 
![image](https://github.com/Lailloken/Lailloken-UI/assets/48123288/0f17776a-2374-4a38-a5c6-601cb6cab018)
![image](https://github.com/Lailloken/Lailloken-UI/assets/48123288/07e88a67-0d3a-43a3-ade0-e2f8f4384dad)

**Comment by SebaFR7474** [comment]
here the act1 route :
#section Act 1
➞ {enter|1_1_3} #The Mud Flats
Find 3x{quest_text|Glyph}
    #sub Little streams connect the 3 Rhoa nests
    #sub Usually form a triangle, but will rarely form a line
➞ {enter|1_1_4_1} #The Submerged Passage
➞ {enter|1_1_4_0} #The Flooded Depths
    #sub Go right of the bridge
Find and kill {kill|The Dweller of the Deep}
    #sub Search opposite the entrance
    #sub Look for the large empty room
    #sub TP Lower prison
➞ {enter|1_1_7_1} #The Lower Prison
➞ {enter|1_1_7_2} #The Upper Prison
➞ {arena|The Warden's Quarters}, kill {kill|Brutus, Lord Incarcerator}
    #sub Recommended Level: 8-10
    #sub Say them to stay before corridor near stairs
    #sub Tp Ship graveyard
➞ {enter|1_1_9} #The Ship Graveyard
    #sub Go down the ledge next to the road
    #sub Far from WP for cave.
➞ {enter|1_1_9a} #The Ship Graveyard Cave    
Find {quest_text|Slave Girl}, take {quest_text|Allflame}
    #sub Search opposite the entrance
➞ {enter|1_1_9}, kill {kill|Captain Fairgraves} #The Ship Graveyard
    #sub TP Marveil Cavern
➞ {enter|1_1_11_1} #The Cavern of Wrath
{waypoint|1_1_11_1} #The Cavern of Wrath
➞ {enter|1_1_11_2} #The Cavern of Anger
    #sub Follow the water
➞ {arena|Merveil's Lair}, kill {kill|Merveil, the Siren}
    #sub Follow the water
    #sub Recommended Level: 11-13

**Comment by Lailloken** [comment]
Can you also upload screenshots of what it looks like within exile-leveling (the normal guide-view, not the edit-view)?

It's already getting late here, I'll have a look tomorrow morning.

**Comment by SebaFR7474** [comment]
![image](https://github.com/Lailloken/Lailloken-UI/assets/48123288/9c1e0e71-2fac-4902-9f12-f92ab23f01d3)

**Comment by SebaFR7474** [comment]
np me too its late almost 23:00 need to go to sleep, thx in advance, so love your tool !

**Comment by Lailloken** [comment, marked as answer]
I have tested the act 1 data you sent me, and it works:
![image](https://github.com/Lailloken/Lailloken-UI/assets/61888437/41c59db4-2924-4e9d-9a7d-8193ee65c34a)

I think you need to reset your guide-progress by long-clicking the ` < ` button. The tool is trying to combine your new, edited guide and some previous progress, but it's not compatible. Whenever you import a guide that differs a lot from the previous one, you have to reset the progress first.

If it still doesn't work after resetting, you can paste the whole exile-leveling export (from clicking 3rd-party export) into a txt-file and upload it here. I'll test it on my end then.
